### PR TITLE
Add initial Contributing proposal

### PR DESCRIPTION
**Pull request by Spokeek**
As i had trouble myself to get started in the project, i wanted to suggest a little contributing doc reminding of the system requirements to get setup for the project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+You can work on any OS system but this repository uses
+
+- Git Submodules
+- Symbolic Links
+
+For Windows machines, you need to setup so your current user can create symbolic links. Refer to this documentation page <https://github.com/git-for-windows/git/wiki/Symbolic-Links>
+
+To setup your project, use the following command to clone with submodules and symbolic links enabled.
+
+`git clone --recurse-submodules -c core.symlinks=true https://github.com/vrc-get/vrc-get`
```
Comment (anatawa12) [inline CONTRIBUTING.md:30]: `-c core.symlinks=true` can be not required on some platform so I think it should be omitted.

In addition, this is complete code so I think it's better to use code block

````suggestion
```bash
git clone --recurse-submodules -c core.symlinks=true https://github.com/vrc-get/vrc-get
```
````

**Review by anatawa12 — APPROVED** [review]
Thank you for suggestion.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Spokeek) [reply CONTRIBUTING.md]: Should be done Now

**Review by anatawa12 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**